src/pkgcheck/checks/python.py

```diff
@@ -77,7 +77,6 @@ class PythonMissingRequiredUse(results.VersionResult, results.Warning):
     def desc(self):
         return 'missing REQUIRED_USE="${PYTHON_REQUIRED_USE}"'
 
-
 class PythonMissingDeps(results.VersionResult, results.Warning):
```

Comment:
Lets define a "class variable" for holding the `dep_value`, meaning:
```python
class PythonMissingDeps(results.VersionResult, results.Warning):
    ...
    dep_value = "PYTHON_DEPS"
```

Then you can use it in function as `self.dep_value`.

And when you inherit the class, you can override this value the same way.

Reply:
Do you still want me to do this now that I'm not (for now?) doing a child class?

Reply:
No, you did it better :)